<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedaClienteView</class>
 <widget class="QDialog" name="SchedaClienteView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>544</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Inria Serif</family>
    <pointsize>11</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Inserisci Scheda Cliente</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>751</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>18</number>
    </property>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Inria Serif</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Scheda Cliente</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditDescrizione"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetImages"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddImage">
        <property name="text">
         <string>Aggiungi Immagine</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
